resources/js/Pages/Results.vue

```diff
@@ -179,7 +179,7 @@
         submitting: boolean
     }
 
-    const SUBMITTING_DELAY_TIME = 1000;
+    const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
Fancy

Reply:
ha, well it's for to make sure that the send function doesn't finish resolving until the loading animation fully completes at least 250ms (before the tests were failing because the setTimeout was finished only after the send promise resolved).